pkg/controller/mysqlbackup/mysqlbackup_controller.go

```diff
@@ -151,6 +151,10 @@ func (r *ReconcileMysqlBackup) Reconcile(ctx context.Context, request reconcile.
 	// set defaults for the backup base on the related cluster
 	backup.SetDefaults(cluster)
 
+	if err = backup.Validate(cluster); err != nil {
```

Comment:
If the validate fails, we will always be blocked here and the user can only find out by looking at the logs.